app/assets/javascripts/map.js

```diff
@@ -84,8 +83,15 @@
       }
 
       App.Map.addInvestmentsMarkers(investmentsMarkers, createMarker);
-      App.Map.addGeozones(map);
+      // TODO: I wonder if it makes sense in the small map when there are only markers
```

Comment:
Layers are certainly useful :+1:. Thanks for adding the feature! :tada:

I'm wondering if it makes sense to disable this feature in some cases, since it's currently shown even when there are no geozones (so only the "markers" layer is there, and so disabling it isn't so useful) and the map shown on the investments is usually pretty small and the icon takes some space :thinking:.

Any thoughts on this?

Reply:
I don't have a strong opinion about this. There is a case where it would still be useful and that is in a multi heading budget, even if geozones are not in use being able to toggle on/off layers for each heading is still quite helpful

Reply:
> even if geozones are not in use being able to toggle on/off layers for each heading is still quite helpful

I'm not sure I'm following :thinking:. If geozones aren't used, what's supposed to happen when you toggle on/off a heading? If I understand correctly, right now turning on/off a heading only changes the visibility of the associated geozone polygon/feature, right? Sorry if I'm missing something! :pray:

Reply:
By the way, I've uploaded a couple of changes.

Since the layers were links to headings or geozones, it was very easy to accidentally click on a link when trying to toggle the visibility of the layer, particularly for people with motor disabilities. So now we're using plain text for the name of a layer.

To avoid the internationalization of the "Markers: " text for now we're not including this layer in the layer control. This also simplifies the logic about the markers layer appearing on the "Create new proposal" form (it no longer appears there).

Finally, in the layers control we're only including geozones with GeoJSON available and we're only adding the layers control if there's more than one geozone with GeoJSON available.

What do you think? :thinking:

Reply:
that all sounds good to me.  I think what I was thinking with the heading layers was that at some point all markers associated with that heading could be toggled on/off enabling you see only the marker locations for a heading even if the heading doesn't have a geozone associated with it

Reply:
> that heading could be toggled on/off enabling you see only the marker locations for a heading even if the heading doesn't have a geozone associated with it

That would be interesting indeed. Future pull requests for this would be welcome :smile:.